internal/engine/experiment/ndt7/ndt7.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	testName    = "ndt"
-	testVersion = "0.8.0"
+	testVersion = "0.9.0"
```

Comment:
Let's bump the version since we've changed the way in which we read messages to honour the context.